autoextract/__main__.py

```diff
@@ -116,6 +118,12 @@ def read_input(input_fp, intype, page_type):
                    choices=["DEBUG", "INFO", "WARNING", "ERROR"],
                    help="log level")
     p.add_argument("--shuffle", help="Shuffle input URLs", action="store_true")
+    p.add_argument("--disable-retries", action="store_true",
+                   help="Disable network, Request-level, and Query-level "
+                        "errors retrying.")
```

Comment:
I'm not sure disabling of network retries makes sense. For example, 429 errors are a part of protocol, and should be handled. What do you think about removing this argument? It still should be possible to disable query-level retries by setting `--max-query-error-retries` to 0.

Reply:
The intention of handle_retries flag in request_raw was mostly to allow implementing retries externally (e.g. with a queue).

Reply:
I was just trying to expose arguments supported by the async client API. It could be useful when testing the percentage of those protocol errors, doesn't it?

Reply:
Makes sense. Removed.